.github/workflows/unit_tests.yml

```
@@ -27,23 +27,29 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
+      # use github python action instead of uv to take advantage of caching
       - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.4)

property "python-version" is not defined in object type {python: number}

(expression)